benchexec/tools/metaval++.py

```diff
@@ -74,15 +84,8 @@ def get_value_from_output(self, output, identifier):
         # only the first group will be returned.
```

Comment:
I do not think this is currently true. I think the code currently always returns the full match.

Reply:
Thanks a lot for your comment!

I improved the documentation